custom_components/weenect/select.py

```diff
@@ -16,8 +16,8 @@
 from .const import DOMAIN, TRACKER_ADDED
 from .entity import WeenectEntity
 
-SELECT_OPTIONS = ("0S", "30S", "1M", "5M", "10M", "30M", "60M")
-DEFAULT_SELECT_OPTION = "30M"
+SELECT_OPTIONS = ("0S", "30S", "1M", "2M", "3M", "5M", "10M", "30M", "60M")
```

Comment:
```suggestion
SELECT_OPTIONS = ("0S", "30S", "1M", "2M", "3M", "5M", "10M")
```

Lets remove the unsupported values